arangod/CMakeLists.txt

```diff
@@ -92,8 +103,12 @@ target_compile_definitions(arango_iresearch
   PUBLIC "$<$<CONFIG:Debug>:IRESEARCH_DEBUG=1>"
 )
 
-set(IRESEARCH_TARGET_DEPENDENCY iresearch-static) # use static to match ArangoDB
-
+get_target_property(arango_iresearch_type arango_iresearch TYPE)
```

Comment:
shared iresearch build is broken and we're considering to drop it completely, please don't use `iresearch-shared` target

Reply:
Okay thanks, will change it.